images/runtime/cni-version.sh

```diff
@@ -1,5 +1,5 @@
 # Code generated by images/scripts/update-cni-version.sh; DO NOT EDIT.
-cni_version="1.1.1"
+cni_version="1.2.0"
```

Comment:
@pchaigno These are changes to the runtime image.

Reply:
Damn, I went twice over the changes and couldn't spot that :facepalm: Thanks and sorry for the noise!